docs/management/api/event-filters-api.asciidoc

```diff
@@ -0,0 +1,596 @@
+:endpoint-artifact-list-id:     endpoint_events
+:endpoint-artifact-name:        Event Filters
+:endpoint-artifact-name-single: Event Filter
```

Comment:
The term "event filters" doesn't need to be capitalized on its own, since it's not a proper name. I think these attributes can be lowercase, because the two below are reserved for special cases where capitalization might be needed to start a sentence or heading.
```suggestion
:endpoint-artifact-name:        event filters
:endpoint-artifact-name-single: event filter
```